cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/hbase/HBaseMetricsTable.java

```diff
@@ -125,7 +134,8 @@ public void rejectedExecution(Runnable r, ThreadPoolExecutor executor) {
                                                  callerRunsPolicy);
 
       this.rowKeyDistributor = new RowKeyDistributorByHashPrefix(
-        new RowKeyDistributorByHashPrefix.OneByteSimpleHash(Constants.Metrics.METRICS_HBASE_SPLITS));
+        new RowKeyDistributorByHashPrefix.
+          OneByteSimpleHash(spec.getIntProperty(Constants.Metrics.METRICS_HBASE_TABLE_SPLITS, 16)));
```

Comment:
1.) If I use cConf value instead of default 16, and if cConf value is modified from 16 (which was 16 in GA), then we will get wrong distributed keys. Should we keep it as 16 only? 
2.) yea the default will be for RC to GA